src/AHitbox.hpp

```diff
@@ -0,0 +1,23 @@
+/*
+** EPITECH PROJECT, 2024
+** OpenthisFuckinDoor
+** File description:
+** AHitbox
+*/
+
+#include "AEntity.hpp"
+
+class AHitbox:public AEntity
+{
+public:
+    AHitbox(float x, float y);
+    ~AHitbox() = default;
+    
+    std::pair<float, float> getHitboxSize() {return (_hitboxSize);}
+    void setHitboxSize(float x, float y) {_hitboxSize = std::pair<float, float>(x, y);}
+    bool isTouched(AHitbox other);
```

Comment:
bool isTouched(AHitbox &other) may be better because you give a reference to another hitbox and not a raw hitbox